sample-dcrdata.conf

```diff
@@ -50,9 +50,10 @@
 ;pg=true
 
 ; PostgreSQL database config (when pg=true)
-; Create a database for each network you plan to run.
-; {netname} will be replaced with the current network name. (e.g. MainNet, TestNet, SimNet)
-;pgdbname=dcrdata_{netname}
+; It's possible to have dcrdata switch between databases based on the network it's connected to. 
```

Comment:
We're not particularly strict in conf files, but a general guideline is to wrap after about 80 columns.